static/css/new_case_studies.css

```diff
@@ -0,0 +1,340 @@
+
+.article {
+  width:100%;
+  border:1px solid grey;
+  display: inline-block;
+  vertical-align: top;
+  position: relative;
+  color:white;
+  font-family:"Roboto", sans-serif;
+  font-weight:300;
+  color:#ffffff;
+  padding-top:8%;
+  padding-bottom:0.5%;
+  padding-left:8%;
+  font-size:1.2em;
+  background:linear-gradient(to right, #2635e4, #c72f6c);
+  background-size:100% auto;
+  background-repeat:no-repeat;
+}
+
+.image {
+
+}
+
+.overlay:before{
+  position: absolute;
+  content:" ";
+  top:0;
+  left:0;
+  width:100%;
+  height:100%;
+  display: block;
+  z-index:0;
+  background:linear-gradient(to right, #2635e4, #c72f6c);
+  opacity:0.8;
+}
+
+
+.article * {
+    position: relative;
+    /* hack */
```

Comment:
hello @alexcontini . Is there additional follow up work to this PR?
It is hard to evaluate the styling, unless there is an example page using these settings. Just curious, how is line 40 used? Otherwise seems okay. Could use a space on:
line 25: .overlay:before{
line 331: @media screen and (max-width: 580px){